website/redirects.js

```diff
@@ -201,7 +201,6 @@ const Redirects=[
   },
   {
     "from": [
-      "/docs/latest/ingestion/flatten-json.html",
```

Comment:
Are we able to keep this redirect, and remove the flatten-json `md` file instead? That way, all the redirects would be kept in one place.

Reply:
It's a tradeoff. We keep it all in one place, but redirects done this way always go to `/latest` since we didn't adopt Docusaurus's versioning feature. The one in the file will stay within the same version. 

I don't have strong feelings about one over the other. Traditionally, I think we've just typically been fine with redirects always going to the latest version